src/extraction/fathom/ruleset_factory.js

```diff
@@ -50,74 +48,79 @@ export default class RulesetFactory {
     // (though we should have distinct penalties for that sort of thing if we
     // care). More importantly, clamp the upper bound of the score so we don't
     // overcome other bonuses and penalties.
-    return trapezoid(area, 80 ** 2, 1000 ** 2) ** this.bigImageCoeff;
+    return linearScale(area, 80 ** 2, 1000 ** 2) ** this.bigImageCoeff;
   }
 
   /** Return whether the computed font size of an element is big. */
   fontIsBig(fnode) {
     const size = parseInt(getComputedStyle(fnode.element).fontSize, 10);
-    return trapezoid(size, 14, 50) ** this.bigFontCoeff;
+    return linearScale(size, 14, 50) ** this.bigFontCoeff;
   }
 
-  /**
-   * Scores fnode with a '$' in its innerText
-   */
+  /** Scores fnode with a '$' in its innerText */
   hasDollarSign(fnode) {
     return (fnode.element.innerText.includes('$') ? ONEISH : ZEROISH) ** this.hasDollarSignCoeff;
   }
 
-  /** Return whether some substring is within a given string, case insensitively. */
-  doesContain(haystack, needle) {
+  /**
+   * Return whether some substring is within a given string, case
+   * insensitively.
+   */
+  caselessIncludes(haystack, needle) {
```

Comment:
omg u didn't

you wrote out the whole word

scandalous

Reply:
I aim to shock. :-)